src/core/connection/Generation.cpp

```diff
@@ -344,6 +344,41 @@ namespace Qv2ray::core::connection
                     inboundsList.append(socksInBoundObject);
                 }
 
+                // TPROXY
```

Comment:
Please format the code using `clang-format` and use our `.clang-format` configuration file.